entities/cpu/spacemit-x60.toml

```diff
@@ -0,0 +1,8 @@
+ruyi-entity = "v0"
+
+related = ["uarch:spacemit-x60"]
+unique_among_type_during_traversal = true
+
+[cpu]
+id = "spacemit-x60"
+display_name = "spacemit-x60"
```

Comment:
This is the human-readable name for display, so it should reflect the marketing name. I think it should be `SpacemiT X60` but please check for yourself.